packages/lesswrong/lib/tableOfContents.ts

```diff
@@ -28,5 +35,241 @@ export interface ToCSectionWithOffset extends ToCSection {
 export interface ToCData {
   html: string | null,
   sections: ToCSection[],
-  headingsCount: number,
```

Comment:
I removed `headingsCount`, because it was redundant with sections.length in the only place it was used (and in some other places appeared to be only set to make the type correct)